backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -194,7 +197,7 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
         |cd $cwd
         |${globManipulations(globFiles)}
         |)
-        |sync
+        |$scriptEpilogue
```

Comment:
Do we want to do the same `replace(...)` trick in case the user has an epilogue script with a pipe that gets eaten by the `stripMargin` ?